web/src/app/alerts/CreateAlertDialog/StepContent/CreateAlertServiceSelect.tsx

```diff
@@ -40,7 +43,7 @@ const query = gql`
   }
 `
 
-const useStyles = makeStyles((theme) => ({
+const useStyles = makeStyles((theme: Theme) => ({
```

Comment:
Let's do typescript conversions in their own PRs

Reply:
Will revert. It made updating easier.